lib/bing-ads-reporting/version.rb

```diff
@@ -1,3 +1,3 @@
 module BingAdsReporting
-  VERSION = "3.0.0"
+  VERSION = '3.0.0'.freeze
```

Comment:
we'll have to change the version for sure

Reply:
done.